modules/tile-converter/src/i3s-attributes-worker.ts

```diff
@@ -15,7 +15,7 @@ export type I3SAttributesWorkerOptions = {
 
 export type B3DMAttributesData = {
   gltfMaterials?: {id: string}[];
-  nodes: any;
+  nodes: any[];
```

Comment:
Any chance we can improve this typing? Or make it unknown?

Reply:
Done